configs/components/puppet-runtime.json

```diff
@@ -1 +1 @@
-{"location":"http://builds.delivery.puppetlabs.net/puppet-runtime/202104070/artifacts/","version":"202104070"}
+{"location":"http://builds.delivery.puppetlabs.net/puppet-runtime/2f8e0e48bfda7f736f1172629acbac03ef83de36/artifacts/","version":"202104070.1.g2f8e0e4"}
```

Comment:
This should be reverted after the new puppet-runtime promotes to pxp-agent-vanagon